framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -7440,13 +7493,11 @@ VkResult VulkanReplayConsumerBase::OverrideCreateAccelerationStructureKHR(
                 capture_id);
         }
 
-        result = device_table->CreateAccelerationStructureKHR(
-            device, &modified_create_info, GetAllocationCallbacks(pAllocator), replay_accel_struct);
+        result = func(device, &modified_create_info, GetAllocationCallbacks(pAllocator), replay_accel_struct);
```

Comment:
This doesn't seem related to the PR purpose - is this and the other changes to the RT calls where you pass in `func` a bugfix unrelated to the block number and marking generated commands?

Reply:
I was looking for injected commands by searching for places where `device_table` is referenced to fetch the api call function pointer. While doing so I discovered these cases where an api function pointer is fetched from `device_table` unnecessarily while the pointer is already available.
 
Strictly speaking this is not 100% related to PR's purposes but, given the opportunity, I clean up those places. I can revert if you insist.

Reply:
No, sounds good to me.  In the future it is slightly helpful to separate clean ups into separate PRs so I can go through those more quickly.